.direnv/flake-inputs/01x5k4nlxcpyd85nnr0b9gm89rm8ff4x-source

```diff
@@ -0,0 +1 @@
+/nix/store/01x5k4nlxcpyd85nnr0b9gm89rm8ff4x-source
```

Comment:
also; what's with all those `.direnv` files?
Are they some sort of cachix thing?

Reply:
> Are they some sort of cachix thing?

No, they are generated by `direnv` which enters an environment when you `cd` into a directory. In this case, we use nix dev shell.

I actually don't know whether I should commit this directory or not; some reddit posts suggest that I should. However, seeing that it contains my local directory, I think I should probably ignore those files from now on :D